src/uct/ib/base/ib_md.c

```diff
@@ -239,6 +239,17 @@ typedef struct {
     struct ibv_mr **mr;
 } uct_ib_md_mem_reg_thread_t;
 
+static void uct_ib_cpuset_move(ucs_cpu_set_t *dst, const ucs_sys_cpuset_t *src)
```

Comment:
- ucs_sys_cpuset_copy(ucs_cpu_set_t *dst, const ucs_sys_cpuset_t *src)
- move this function to sys.h/sys.c